todo_app/app.py

```diff
@@ -1,11 +1,37 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
+from todo_app.data.session_items import SessionItems
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
-
+si = SessionItems()
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items=si.get_items()
+    items = sorted(items, key=lambda x: x['status'], reverse=True)
+    return render_template("index.html", items=items)
+
+@app.route('/add-item', methods=["GET", "POST"])
+def add_item():
+    if request.method == "POST":
+        title = request.form.get("title", "")
+        si.add_item(title)
+        return redirect('/')
+    return render_template("add_item.html")
```

Comment:
Personal preference rather than strong recommendation, but I quite like splitting these methods up clearly rather than combining them - it'll help keep them shorter if either/both method starts to get more complex:
```suggestion
@app.route('/add-item', methods=["GET"])
def add_item_page():
    return render_template("add_item.html")

@app.route('/add-item', methods=["POST"])
def add_item():
    title = request.form.get("title", "")
    si.add_item(title)
    return redirect('/')
```